src/visualization/efmviz/selectGeneFromGPR_efmviz.m

```diff
@@ -0,0 +1,76 @@
+function expressionRxns = selectGeneFromGPR_efmviz(model, gene_names, gene_exp, gene_sig, parsedGPR, minSum)
```

Comment:
Please incorporate the adaptions into the file this one is derived from and make them optional to avoid duplication (and thus double maintenance) of code.